src/docs/primitives/a-model.md

```diff
@@ -29,3 +29,9 @@ Using scale to increase to size of the loaded tree model:
 ```html
 <a-model scale="10 10 10" src="https://aframe.io/aframe/examples/_models/tree1/tree1.dae"></a-model>
 ```
```

Comment:
added links to the table. we don't have anchors for the table and table rows yet, but we should later. this look okay to you?